tests/D2L.CodeStyle.Analyzers.Test/Specs/ImmutabilityAnalyzer.cs

```diff
@@ -74,6 +74,26 @@ class ClassWithAnnotatedMutableStateDoesntFail : IImmutable {
 		}
 	}
 
+	class ExternalMarkedImmutableTests {
+		class /* ImmutableClassIsnt('m_bad' is not read-only) */ MutableEncoding /**/ : System.Text.Encoding {
+			private int m_bad;
+		}
+
+		class AuditedMutableEncoding : System.Text.Encoding {

```

Comment:
I like that this isn't `sealed`, tests some more stuff, subtly.

Reply:
(adding to the list of built-in `[Immutable]` types behaves like `[Immutable]`. Things that are `[Immutable]` (even implicitly) aren't required to be `sealed`.)